tests/acceptance/features/webUISharingInternalUsersToRoot/shareWithUsers.feature

```diff
@@ -52,7 +55,8 @@ Feature: Sharing files and folders with internal users
 
 
   Scenario: share a file with another internal user who overwrites and unshares the file
-    Given user "Brian" has logged in using the webUI
+    Given user "Brian" has created file "lorem.txt"
+    And user "Brian" has logged in using the webUI
     And user "Brian" has renamed file "lorem.txt" to "new-lorem.txt"
```

Comment:
I think you could just upload a file called `new-lorem.txt` and not bother renaming it